packages/flutter/lib/src/services/message_codec.dart

```diff
@@ -26,6 +26,62 @@ abstract class MessageCodec<T> {
   T decodeMessage(ByteData message);
 }
 
+/// An command object representing the invocation of a named method.
+class MethodCall {
+  final String method;
+  final dynamic arguments;
+  
+  /// Creates a [MethodCall] representing the invocation of [method] with the
+  /// specified [arguments].
+  MethodCall(this.method, [this.arguments]) {
+    assert(method != null);
+  }
+  
+  @override
+  bool operator== (dynamic o) {
+    if (o is! MethodCall)
```

Comment:
this should probably be a runtimeType check so that `==` isn't sensitive to the order of its arguments